### PR TITLE
README: import "current status"

### PR DESCRIPTION
**Pull request by mistydemeo**
This is adapted from @xu-cheng's table in #1, updated to reflect the presence of 10.4 builds.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,27 @@ docker run -t -i homebrew-portable /bin/bash
 brew portable-package <formula>
 ```
 
+## Current status
+
+| Formula | OS X 10.12 | OS X 10.5 | OS X 10.4 | Linuxbrew[0] |
+| :-: | :-: | :-: | :-: | :-: |
+| ncurses | N/A | N/A | N/A | :white_check_mark: |
+| expat | N/A | N/A | N/A | :white_check_mark: |
```
Comment (xu-cheng) [inline README.md]: `portable-expat` is used in OS X 10.4. So it should not be `N/A`. right?

File: README.md [hunk 2]
```diff
@@ -16,6 +16,27 @@ docker run -t -i homebrew-portable /bin/bash
 brew portable-package <formula>
 ```
 
+## Current status
+
+| Formula | OS X 10.12 | OS X 10.5 | OS X 10.4 | Linuxbrew[0] |
```
Comment (xu-cheng) [inline README.md]: Maybe we should call OS X 10.12 as macOS 10.12

File: README.md [hunk 3]
```diff
@@ -16,6 +16,27 @@ docker run -t -i homebrew-portable /bin/bash
...
+| Ruby | :white_check_mark::warning: (64 bit only)[1] | :white_check_mark::warning: (32/64 bit only)[1] | :white_check_mark: (Single arch only)[1] | :white_check_mark::warning: [3] |
+| Curl | :white_check_mark: | :white_check_mark: | :white_check_mark: | :x: [2] |
+| Git | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: |
+
+macOS/OS X builds on 10.6 and newer target 32-bit/64-bit Intel Macs. OS X builds on 10.4 and 10.5 target 32-bit PowerPC and Intel Macs.
+
+[0]: Linuxbrew is 64 bit only, which should be built in CentOS docker using system libraries and compiler.
+[1]: Fail on universal build. https://gist.github.com/cb5e6b142c39116dbe0b954885f1054e. It appears to be Ruby's bug, as it's fixed for Ruby 2.1.
+[2]: Binary files report segment fault when running on CentOS 6(work fine on CentOS 5)
+[3]: Linux build `irb` seems to fail to link staticly to ncurses. If `portable-ncurses` is removed, `irb` will fail to handle left, right or backspace keystroke.
```
Comment (xu-cheng) [inline README.md]: It seems that this list is not formatted proabably in markdown.

Comment (mistydemeo) [inline README.md]: Fixed!

**Pull request merged**